packages/opentelemetry-metrics/src/export/types.ts

```diff
@@ -30,13 +31,6 @@ export type Sum = number;
 /** LastValue returns last value. */
 export type LastValue = number;
 
-export interface Distribution {
```

Comment:
IMO we should keep all the point value types together. Currently, `Sum` and `LastValue` are in `export/types`, `Distribution` is in `measureexact.ts` and `Checkpoint` is in `histogram.ts`. WDYT?